appservice/src/main/java/nu/yona/server/device/rest/DeviceController.java

```diff
@@ -136,6 +155,39 @@
 		}
 	}
 
+	@RequestMapping(value = "/{deviceId}/apple.mobileconfig", method = RequestMethod.GET)
+	@ResponseBody
+	public ResponseEntity<byte[]> getAppleMobileConfig(
+			@RequestHeader(value = Constants.PASSWORD_HEADER) Optional<String> password, @PathVariable UUID userId,
+			@PathVariable UUID deviceId)
+	{
+		HttpHeaders headers = new HttpHeaders();
+		headers.setContentType(new MediaType("application", "x-apple-aspen-config"));
+		try (CryptoSession cryptoSession = CryptoSession.start(password, () -> userService.canAccessPrivateData(userId)))
+		{
+			return new ResponseEntity<>(getUserSpecificAppleMobileConfig(deviceService.getDevice(deviceId)), headers,
+					HttpStatus.OK);
+		}
+	}
+
+	private byte[] getUserSpecificAppleMobileConfig(UserDeviceDto device)
```

Comment:
Rename to `getDeviceSpecificAppleMobileConfig`

Reply:
Done